src/Illuminate/Database/Query/Grammars/SqlServerGrammar.php

```diff
@@ -154,6 +154,17 @@ protected function compileJsonContains($column, $value)
         return $value.' in (select [value] from openjson('.$field.$path.'))';
     }
 
+    /**
+     * Compile a "JSON cast" statement into SQL.
+     *
+     * @param  string  $value
+     * @return string
+     */
+    public function compileJsonCast($value)
+    {
+        return "json_query($value)";
```

Comment:
for code consistency, I would switch to single quotes and concatenation, rather than double quotes and interpolation.

Reply:
Noted! Will update later tonight 👍